CI: v1.10: Cilium L4LB XDP (ci-l4lb-1.10): error: timed out waiting for the condition on pods/cilium-fxnz4

PRs on the v1.10 branch are failing the l4lb test with the following error while setting up the test.

Example run:
https://github.com/cilium/cilium/runs/4477255000?check_suite_focus=true

No artifacts were gathered as the workflow is not set up to correctly gather a sysdump.

```
Run ssh default "sudo /bin/sh -c 'cd /vagrant/test/l4lb && ./test.sh cilium eeee50189b1df5f33c801ddc7c3cd25ca3a5facf'"
  ssh default "sudo /bin/sh -c 'cd /vagrant/test/l4lb && ./test.sh cilium eeee50189b1df5f33c801ddc7c3cd25ca3a5facf'"
  shell: /bin/bash -e {0}
  env:
    check_url: https://github.com/cilium/cilium/actions/runs/1561039026
+ IMG_OWNER=cilium
+ IMG_TAG=eeee50189b1df5f33c801ddc7c3cd25ca3a5facf
+ clang -O2 -Wall -target bpf -c bpf_xdp_veth_host.c -o bpf_xdp_veth_host.o
+ clang -O2 -Wall -target bpf -c test_tc_tunnel.c -o test_tc_tunnel.o
test_tc_tunnel.c:456:7: warning: unused variable 'buf' [-Wunused-variable]
        char buf[sizeof(struct v6hdr)];
             ^
1 warning generated.
+ kind create cluster --config kind-config.yaml
Creating cluster "kind" ...
 • Ensuring node image (kindest/node:v1.21.1) 🖼  ...
 ✓ Ensuring node image (kindest/node:v1.21.1) 🖼
 • Preparing nodes 📦 📦   ...
 ✓ Preparing nodes 📦 📦 
 • Writing configuration 📜  ...
 ✓ Writing configuration 📜
 • Starting control-plane 🕹️  ...
 ✓ Starting control-plane 🕹️
 • Installing StorageClass 💾  ...
 ✓ Installing StorageClass 💾
 • Joining worker nodes 🚜  ...
 ✓ Joining worker nodes 🚜
Set kubectl context to "kind-kind"
You can now use your cluster with:

kubectl cluster-info --context kind-kind

Have a question, bug, or feature request? Let us know! https://kind.sigs.k8s.io/#community 🙂
+ ip l a l4lb-veth0 type veth peer l4lb-veth1
+ SECOND_LB_NODE_IP=3.3.3.2
+ ip a a 3.3.3.1/24 dev l4lb-veth0
++ docker inspect kind-control-plane -f '{{ .State.Pid }}'
+ CONTROL_PLANE_PID=12805
+ ip l s dev l4lb-veth1 netns 12805
+ ip l s dev l4lb-veth0 up
+ nsenter -t 12805 -n /bin/sh -c '    ip a a 3.3.3.2/24 dev l4lb-veth1 &&     ip l s dev l4lb-veth1 up'
+ helm install cilium ../../install/kubernetes/cilium --wait --namespace kube-system --set image.repository=quay.io/cilium/cilium-ci --set image.tag=eeee50189b1df5f33c801ddc7c3cd25ca3a5facf --set image.useDigest=false --set image.pullPolicy=IfNotPresent --set operator.enabled=false --set loadBalancer.standalone=true --set loadBalancer.algorithm=maglev --set loadBalancer.mode=dsr --set loadBalancer.acceleration=native --set loadBalancer.dsrDispatch=ipip --set 'devices={eth0,l4lb-veth1}' --set nodePort.directRoutingDevice=eth0 --set ipv6.enabled=false --set 'affinity.nodeAffinity.requiredDuringSchedulingIgnoredDuringExecution.nodeSelectorTerms[0].matchExpressions[0].key=kubernetes.io/hostname' --set 'affinity.nodeAffinity.requiredDuringSchedulingIgnoredDuringExecution.nodeSelectorTerms[0].matchExpressions[0].operator=In' --set 'affinity.nodeAffinity.requiredDuringSchedulingIgnoredDuringExecution.nodeSelectorTerms[0].matchExpressions[0].values[0]=kind-control-plane'
NAME: cilium
LAST DEPLOYED: Thu Dec  9 23:04:50 2021
NAMESPACE: kube-system
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
You have successfully installed Cilium with Hubble.

Your release version is 1.10.5.

For any further help, visit https://docs.cilium.io/en/v1.10/gettinghelp
++ docker exec -i kind-control-plane /bin/sh -c 'echo $(( $(ip -o l show eth0 | awk "{print $1}" | cut -d: -f1) ))'
+ IFIDX=7
++ cut -d@ -f1
++ awk '{print $2}'
++ grep if7
++ ip -o l
+ LB_VETH_HOST=vethc329158
+ ip l set dev vethc329158 xdp obj bpf_xdp_veth_host.o
+ ip l set dev l4lb-veth0 xdp obj bpf_xdp_veth_host.o
+ ethtool -K vethc329158 rx off tx off
Actual changes:
tx-checksum-ip-generic: off
tx-tcp-segmentation: off [not requested]
+ ethtool -K l4lb-veth0 rx off tx off
tx-tcp-ecn-segmentation: off [not requested]
tx-tcp-mangleid-segmentation: off [not requested]
tx-tcp6-segmentation: off [not requested]
tx-checksum-sctp: off
rx-checksum: off
Actual changes:
tx-checksum-ip-generic: off
tx-tcp-segmentation: off [not requested]
tx-tcp-ecn-segmentation: off [not requested]
tx-tcp-mangleid-segmentation: off [not requested]
tx-tcp6-segmentation: off [not requested]
tx-checksum-sctp: off
rx-checksum: off
+ docker exec kind-worker /bin/sh -c 'apt-get update && apt-get install -y nginx && systemctl start nginx'
Get:1 http://archive.ubuntu.com/ubuntu groovy InRelease [267 kB]
Get:2 http://security.ubuntu.com/ubuntu groovy-security InRelease [110 kB]
Get:3 http://archive.ubuntu.com/ubuntu groovy-updates InRelease [115 kB]
Get:4 http://archive.ubuntu.com/ubuntu groovy-backports InRelease [101 kB]
Get:5 http://security.ubuntu.com/ubuntu groovy-security/restricted amd64 Packages [99.0 kB]
Get:6 http://security.ubuntu.com/ubuntu groovy-security/universe amd64 Packages [429 kB]
Get:7 http://archive.ubuntu.com/ubuntu groovy/restricted amd64 Packages [87.5 kB]
Get:8 http://archive.ubuntu.com/ubuntu groovy/multiverse amd64 Packages [247 kB]
Get:9 http://archive.ubuntu.com/ubuntu groovy/universe amd64 Packages [16.1 MB]
Get:10 http://security.ubuntu.com/ubuntu groovy-security/main amd64 Packages [284 kB]
Get:11 http://security.ubuntu.com/ubuntu groovy-security/multiverse amd64 Packages [12.9 kB]
Get:12 http://archive.ubuntu.com/ubuntu groovy/main amd64 Packages [1768 kB]
Get:13 http://archive.ubuntu.com/ubuntu groovy-updates/multiverse amd64 Packages [20.1 kB]
Get:14 http://archive.ubuntu.com/ubuntu groovy-updates/restricted amd64 Packages [110 kB]
Get:15 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 Packages [453 kB]
Get:16 http://archive.ubuntu.com/ubuntu groovy-updates/universe amd64 Packages [559 kB]
Get:17 http://archive.ubuntu.com/ubuntu groovy-backports/main amd64 Packages [2681 B]
Get:18 http://archive.ubuntu.com/ubuntu groovy-backports/universe amd64 Packages [6237 B]
Fetched 20.8 MB in 5s (3893 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  fontconfig-config fonts-dejavu-core libexpat1 libfontconfig1 libfreetype6
  libgd3 libicu67 libjbig0 libjpeg-turbo8 libjpeg8 libmaxminddb0
  libnginx-mod-http-image-filter libnginx-mod-http-xslt-filter
  libnginx-mod-mail libnginx-mod-stream libnginx-mod-stream-geoip2 libpng16-16
  libtiff5 libwebp6 libx11-6 libx11-data libxau6 libxcb1 libxdmcp6 libxml2
  libxpm4 libxslt1.1 nginx-common nginx-core
Suggested packages:
  libgd-tools mmdb-bin fcgiwrap nginx-doc ssl-cert
The following NEW packages will be installed:
  fontconfig-config fonts-dejavu-core libexpat1 libfontconfig1 libfreetype6
  libgd3 libicu67 libjbig0 libjpeg-turbo8 libjpeg8 libmaxminddb0
  libnginx-mod-http-image-filter libnginx-mod-http-xslt-filter
  libnginx-mod-mail libnginx-mod-stream libnginx-mod-stream-geoip2 libpng16-16
  libtiff5 libwebp6 libx11-6 libx11-data libxau6 libxcb1 libxdmcp6 libxml2
  libxpm4 libxslt1.1 nginx nginx-common nginx-core
0 upgraded, 30 newly installed, 0 to remove and 20 not upgraded.
Need to get 13.2 MB of archives.
After this operation, 49.4 MB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu groovy/main amd64 libexpat1 amd64 2.2.9-1build1 [73.3 kB]
Get:2 http://archive.ubuntu.com/ubuntu groovy/main amd64 libicu67 amd64 67.1-4 [8620 kB]
Get:3 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 libxml2 amd64 2.9.10+dfsg-5ubuntu0.20.10.2 [624 kB]
Get:4 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 libmaxminddb0 amd64 1.4.2-0ubuntu1.20.10.1 [26.2 kB]
Get:5 http://archive.ubuntu.com/ubuntu groovy/main amd64 libpng16-16 amd64 1.6.37-3 [181 kB]
Get:6 http://archive.ubuntu.com/ubuntu groovy/main amd64 libxau6 amd64 1:1.0.9-0ubuntu1 [7488 B]
Get:7 http://archive.ubuntu.com/ubuntu groovy/main amd64 libxdmcp6 amd64 1:1.1.3-0ubuntu1 [10.6 kB]
Get:8 http://archive.ubuntu.com/ubuntu groovy/main amd64 libxcb1 amd64 1.14-2 [44.7 kB]
Get:9 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 libx11-data all 2:1.6.12-1ubuntu0.1 [113 kB]
Get:10 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 libx11-6 amd64 2:1.6.12-1ubuntu0.1 [575 kB]
Get:11 http://archive.ubuntu.com/ubuntu groovy/main amd64 fonts-dejavu-core all 2.37-2 [1041 kB]
Get:12 http://archive.ubuntu.com/ubuntu groovy/main amd64 fontconfig-config all 2.13.1-2ubuntu3 [28.8 kB]
Get:13 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 libfreetype6 amd64 2.10.2+dfsg-3ubuntu1 [349 kB]
Get:14 http://archive.ubuntu.com/ubuntu groovy/main amd64 libfontconfig1 amd64 2.13.1-2ubuntu3 [114 kB]
Get:15 http://archive.ubuntu.com/ubuntu groovy/main amd64 libjpeg-turbo8 amd64 2.0.3-0ubuntu2 [118 kB]
Get:16 http://archive.ubuntu.com/ubuntu groovy/main amd64 libjpeg8 amd64 8c-2ubuntu8 [2194 B]
Get:17 http://archive.ubuntu.com/ubuntu groovy/main amd64 libjbig0 amd64 2.1-3.1build1 [26.7 kB]
Get:18 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 libwebp6 amd64 0.6.1-2ubuntu0.20.10.1 [183 kB]
Get:19 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 libtiff5 amd64 4.1.0+git191117-2ubuntu0.20.10.1 [164 kB]
Get:20 http://archive.ubuntu.com/ubuntu groovy/main amd64 libxpm4 amd64 1:3.5.12-1 [34.0 kB]
Get:21 http://archive.ubuntu.com/ubuntu groovy/main amd64 libgd3 amd64 2.3.0-2 [118 kB]
Get:22 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 nginx-common all 1.18.0-6ubuntu2.2 [39.4 kB]
Get:23 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 libnginx-mod-http-image-filter amd64 1.18.0-6ubuntu2.2 [15.8 kB]
Get:24 http://archive.ubuntu.com/ubuntu groovy/main amd64 libxslt1.1 amd64 1.1.34-4 [152 kB]
Get:25 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 libnginx-mod-http-xslt-filter amd64 1.18.0-6ubuntu2.2 [14.1 kB]
Get:26 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 libnginx-mod-mail amd64 1.18.0-6ubuntu2.2 [43.7 kB]
Get:27 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 libnginx-mod-stream amd64 1.18.0-6ubuntu2.2 [68.1 kB]
Get:28 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 libnginx-mod-stream-geoip2 amd64 1.18.0-6ubuntu2.2 [9648 B]
Get:29 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 nginx-core amd64 1.18.0-6ubuntu2.2 [427 kB]
Get:30 http://archive.ubuntu.com/ubuntu groovy-updates/main amd64 nginx amd64 1.18.0-6ubuntu2.2 [3988 B]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 13.2 MB in 2s (5622 kB/s)
Selecting previously unselected package libexpat1:amd64.
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 6470 files and directories currently installed.)
Preparing to unpack .../00-libexpat1_2.2.9-1build1_amd64.deb ...
Unpacking libexpat1:amd64 (2.2.9-1build1) ...
Selecting previously unselected package libicu67:amd64.
Preparing to unpack .../01-libicu67_67.1-4_amd64.deb ...
Unpacking libicu67:amd64 (67.1-4) ...
Selecting previously unselected package libxml2:amd64.
Preparing to unpack .../02-libxml2_2.9.10+dfsg-5ubuntu0.20.10.2_amd64.deb ...
Unpacking libxml2:amd64 (2.9.10+dfsg-5ubuntu0.20.10.2) ...
Selecting previously unselected package libmaxminddb0:amd64.
Preparing to unpack .../03-libmaxminddb0_1.4.2-0ubuntu1.20.10.1_amd64.deb ...
Unpacking libmaxminddb0:amd64 (1.4.2-0ubuntu1.20.10.1) ...
Selecting previously unselected package libpng16-16:amd64.
Preparing to unpack .../04-libpng16-16_1.6.37-3_amd64.deb ...
Unpacking libpng16-16:amd64 (1.6.37-3) ...
Selecting previously unselected package libxau6:amd64.
Preparing to unpack .../05-libxau6_1%3a1.0.9-0ubuntu1_amd64.deb ...
Unpacking libxau6:amd64 (1:1.0.9-0ubuntu1) ...
Selecting previously unselected package libxdmcp6:amd64.
Preparing to unpack .../06-libxdmcp6_1%3a1.1.3-0ubuntu1_amd64.deb ...
Unpacking libxdmcp6:amd64 (1:1.1.3-0ubuntu1) ...
Selecting previously unselected package libxcb1:amd64.
Preparing to unpack .../07-libxcb1_1.14-2_amd64.deb ...
Unpacking libxcb1:amd64 (1.14-2) ...
Selecting previously unselected package libx11-data.
Preparing to unpack .../08-libx11-data_2%3a1.6.12-1ubuntu0.1_all.deb ...
Unpacking libx11-data (2:1.6.12-1ubuntu0.1) ...
Selecting previously unselected package libx11-6:amd64.
Preparing to unpack .../09-libx11-6_2%3a1.6.12-1ubuntu0.1_amd64.deb ...
Unpacking libx11-6:amd64 (2:1.6.12-1ubuntu0.1) ...
Selecting previously unselected package fonts-dejavu-core.
Preparing to unpack .../10-fonts-dejavu-core_2.37-2_all.deb ...
Unpacking fonts-dejavu-core (2.37-2) ...
Selecting previously unselected package fontconfig-config.
Preparing to unpack .../11-fontconfig-config_2.13.1-2ubuntu3_all.deb ...
Unpacking fontconfig-config (2.13.1-2ubuntu3) ...
Selecting previously unselected package libfreetype6:amd64.
Preparing to unpack .../12-libfreetype6_2.10.2+dfsg-3ubuntu1_amd64.deb ...
Unpacking libfreetype6:amd64 (2.10.2+dfsg-3ubuntu1) ...
Selecting previously unselected package libfontconfig1:amd64.
Preparing to unpack .../13-libfontconfig1_2.13.1-2ubuntu3_amd64.deb ...
Unpacking libfontconfig1:amd64 (2.13.1-2ubuntu3) ...
Selecting previously unselected package libjpeg-turbo8:amd64.
Preparing to unpack .../14-libjpeg-turbo8_2.0.3-0ubuntu2_amd64.deb ...
Unpacking libjpeg-turbo8:amd64 (2.0.3-0ubuntu2) ...
Selecting previously unselected package libjpeg8:amd64.
Preparing to unpack .../15-libjpeg8_8c-2ubuntu8_amd64.deb ...
Unpacking libjpeg8:amd64 (8c-2ubuntu8) ...
Selecting previously unselected package libjbig0:amd64.
Preparing to unpack .../16-libjbig0_2.1-3.1build1_amd64.deb ...
Unpacking libjbig0:amd64 (2.1-3.1build1) ...
Selecting previously unselected package libwebp6:amd64.
Preparing to unpack .../17-libwebp6_0.6.1-2ubuntu0.20.10.1_amd64.deb ...
Unpacking libwebp6:amd64 (0.6.1-2ubuntu0.20.10.1) ...
Selecting previously unselected package libtiff5:amd64.
Preparing to unpack .../18-libtiff5_4.1.0+git191117-2ubuntu0.20.10.1_amd64.deb ...
Unpacking libtiff5:amd64 (4.1.0+git191117-2ubuntu0.20.10.1) ...
Selecting previously unselected package libxpm4:amd64.
Preparing to unpack .../19-libxpm4_1%3a3.5.12-1_amd64.deb ...
Unpacking libxpm4:amd64 (1:3.5.12-1) ...
Selecting previously unselected package libgd3:amd64.
Preparing to unpack .../20-libgd3_2.3.0-2_amd64.deb ...
Unpacking libgd3:amd64 (2.3.0-2) ...
Selecting previously unselected package nginx-common.
Preparing to unpack .../21-nginx-common_1.18.0-6ubuntu2.2_all.deb ...
Unpacking nginx-common (1.18.0-6ubuntu2.2) ...
Selecting previously unselected package libnginx-mod-http-image-filter.
Preparing to unpack .../22-libnginx-mod-http-image-filter_1.18.0-6ubuntu2.2_amd64.deb ...
Unpacking libnginx-mod-http-image-filter (1.18.0-6ubuntu2.2) ...
Selecting previously unselected package libxslt1.1:amd64.
Preparing to unpack .../23-libxslt1.1_1.1.34-4_amd64.deb ...
Unpacking libxslt1.1:amd64 (1.1.34-4) ...
Selecting previously unselected package libnginx-mod-http-xslt-filter.
Preparing to unpack .../24-libnginx-mod-http-xslt-filter_1.18.0-6ubuntu2.2_amd64.deb ...
Unpacking libnginx-mod-http-xslt-filter (1.18.0-6ubuntu2.2) ...
Selecting previously unselected package libnginx-mod-mail.
Preparing to unpack .../25-libnginx-mod-mail_1.18.0-6ubuntu2.2_amd64.deb ...
Unpacking libnginx-mod-mail (1.18.0-6ubuntu2.2) ...
Selecting previously unselected package libnginx-mod-stream.
Preparing to unpack .../26-libnginx-mod-stream_1.18.0-6ubuntu2.2_amd64.deb ...
Unpacking libnginx-mod-stream (1.18.0-6ubuntu2.2) ...
Selecting previously unselected package libnginx-mod-stream-geoip2.
Preparing to unpack .../27-libnginx-mod-stream-geoip2_1.18.0-6ubuntu2.2_amd64.deb ...
Unpacking libnginx-mod-stream-geoip2 (1.18.0-6ubuntu2.2) ...
Selecting previously unselected package nginx-core.
Preparing to unpack .../28-nginx-core_1.18.0-6ubuntu2.2_amd64.deb ...
Unpacking nginx-core (1.18.0-6ubuntu2.2) ...
Selecting previously unselected package nginx.
Preparing to unpack .../29-nginx_1.18.0-6ubuntu2.2_amd64.deb ...
Unpacking nginx (1.18.0-6ubuntu2.2) ...
Setting up libexpat1:amd64 (2.2.9-1build1) ...
Setting up libxau6:amd64 (1:1.0.9-0ubuntu1) ...
Setting up libxdmcp6:amd64 (1:1.1.3-0ubuntu1) ...
Setting up libxcb1:amd64 (1.14-2) ...
Setting up libicu67:amd64 (67.1-4) ...
Setting up libmaxminddb0:amd64 (1.4.2-0ubuntu1.20.10.1) ...
Setting up nginx-common (1.18.0-6ubuntu2.2) ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Readline
debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.30.3 /usr/local/share/perl/5.30.3 /usr/lib/x86_64-linux-gnu/perl5/5.30 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl-base /usr/lib/x86_64-linux-gnu/perl/5.30 /usr/share/perl/5.30 /usr/local/lib/site_perl) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
debconf: falling back to frontend: Teletype
Created symlink /etc/systemd/system/multi-user.target.wants/nginx.service → /lib/systemd/system/nginx.service.
/usr/sbin/policy-rc.d returned 101, not running 'start nginx.service'
Setting up libjbig0:amd64 (2.1-3.1build1) ...
Setting up libx11-data (2:1.6.12-1ubuntu0.1) ...
Setting up libpng16-16:amd64 (1.6.37-3) ...
Setting up libwebp6:amd64 (0.6.1-2ubuntu0.20.10.1) ...
Setting up fonts-dejavu-core (2.37-2) ...
Setting up libjpeg-turbo8:amd64 (2.0.3-0ubuntu2) ...
Setting up libx11-6:amd64 (2:1.6.12-1ubuntu0.1) ...
Setting up libxml2:amd64 (2.9.10+dfsg-5ubuntu0.20.10.2) ...
Setting up libjpeg8:amd64 (8c-2ubuntu8) ...
Setting up libnginx-mod-mail (1.18.0-6ubuntu2.2) ...
Setting up libxpm4:amd64 (1:3.5.12-1) ...
Setting up fontconfig-config (2.13.1-2ubuntu3) ...
Setting up libnginx-mod-stream (1.18.0-6ubuntu2.2) ...
Setting up libfreetype6:amd64 (2.10.2+dfsg-3ubuntu1) ...
Setting up libxslt1.1:amd64 (1.1.34-4) ...
Setting up libtiff5:amd64 (4.1.0+git191117-2ubuntu0.20.10.1) ...
Setting up libfontconfig1:amd64 (2.13.1-2ubuntu3) ...
Setting up libnginx-mod-stream-geoip2 (1.18.0-6ubuntu2.2) ...
Setting up libnginx-mod-http-xslt-filter (1.18.0-6ubuntu2.2) ...
Setting up libgd3:amd64 (2.3.0-2) ...
Setting up libnginx-mod-http-image-filter (1.18.0-6ubuntu2.2) ...
Setting up nginx-core (1.18.0-6ubuntu2.2) ...
invoke-rc.d: policy-rc.d denied execution of start.
Setting up nginx (1.18.0-6ubuntu2.2) ...
Processing triggers for systemd (246.6-1ubuntu1.3) ...
Processing triggers for libc-bin (2.32-0ubuntu3) ...
++ docker exec kind-worker ip -o -4 a s eth0
++ cut -d/ -f1
++ awk '{print $4}'
+ WORKER_IP=172.18.0.2
++ docker inspect kind-worker -f '{{ .State.Pid }}'
+ nsenter -t 12803 -n /bin/sh -c 'tc qdisc add dev eth0 clsact && tc filter add dev eth0 ingress bpf direct-action object-file ./test_tc_tunnel.o section decap'
++ kubectl -n kube-system get pod -l k8s-app=cilium '-o=jsonpath={.items[0].metadata.name}'
+ CILIUM_POD_NAME=cilium-fxnz4
+ kubectl -n kube-system wait --for=condition=Ready pod cilium-fxnz4 --timeout=5m
error: timed out waiting for the condition on pods/cilium-fxnz4
Error: Process completed with exit code 1.
1s
Run scp default:/tmp/cilium-sysdump-out.zip .
scp: /tmp/cilium-sysdump-out.zip: No such file or directory
Error: Process completed with exit code 1.
```

--- a/daemon/cmd/kube_proxy_replacement.go
+++ b/daemon/cmd/kube_proxy_replacement.go
@@ -628,10 +628,6 @@ func finishKubeProxyReplacementInit(isKubeProxyReplacementStrict bool) error {
 		}
 	}
 
-	if option.Config.NodePortAcceleration != option.NodePortAccelerationDisabled &&
-		len(option.Config.Devices) != 1 {
-		return fmt.Errorf("Cannot set NodePort acceleration due to multi-device setup (%q). Specify --%s with a single device to enable NodePort acceleration.", option.Config.Devices, option.Devices)
-	}
 	return nil
 }
 

